src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
+spring.datasource.url=jdbc:postgresql://${APP_DB_HOST}:${APP_DB_PORT}/${APP_DB_NAME}
```

Comment:
I think that this line with parameters would be more or less standard, so it wasn't necessary to make environment variables for it.